domains/Settings/SettingsHome/index.js

```diff
@@ -0,0 +1,144 @@
+              <Text style={styles.text}>{I18n.t('accountSettings.changePassword')}</Text>
+              <IconButton
+                icon="chevron-right"
+                size={30}
+                color={theme.colors.primary}
+                style={{ marginLeft: 'auto', marginTop: -5, marginBottom: -10 }}
+                onPress={() => {
+                  setAccountSettingsView('ChangePassword');
+                }}
+              />
+            </View>
+            <View style={styles.horizontalLineGray} />
+            <View style={{ flexDirection: 'row' }}>
+              <Text style={styles.text}>{I18n.t('accountSettings.findRecords')}</Text>
+              <IconButton
+                icon="chevron-right"
+                size={30}
+                color={theme.colors.primary}
+                style={{ marginLeft: 'auto', marginTop: -5, marginBottom: -10 }}
+                onPress={() => {
+                  setAccountSettingsView('FindRecords');
+                }}
+              />
+            </View>
+            <View style={styles.horizontalLineGray} />
+            <View style={{ flexDirection: 'row' }}>
+              <Text style={styles.text}>{I18n.t('accountSettings.language')}</Text>
+              <IconButton
+                icon="chevron-right"
+                size={30}
+                color={theme.colors.primary}
+                style={{ marginLeft: 'auto', marginTop: -5, marginBottom: -10 }}
+                onPress={() => {
+                  setAccountSettingsView('Language');
+                }}
+              />
+            </View>
+            <View style={styles.horizontalLineGray} />
+          </View>
+          <Button onPress={() => {
+            setView(prevView);
+          }}
+          >
+            {I18n.t('accountSettings.back')}
+          </Button>
+          <Button mode="contained" onPress={logOut} style={{ marginTop: 20 }}>{I18n.t('accountSettings.logout')}</Button>
+        </View>
+      )}
+      {accountSettingsView !== '' && (
+        <View>
+          <AccountSettings
+            settingsView={settingsView}
+            setSettingsView={setSettingsView}
+            accountSettingsView={accountSettingsView}
+            setAccountSettingsView={setAccountSettingsView}
+          />
+        </View>
+      )}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  cardSmallStyle: {
+    height: 110,
+    width: 150,
+    marginHorizontal: 7,
+    marginVertical: 7,
+  },
+  svg: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  cardContainer: {
+    alignItems: 'center',
+    marginHorizontal: 14,
+    marginVertical: 14,
+  },
+  textContainer: {
+    flexDirection: 'row'
+  },
+  text: {
+    flexShrink: 1,
+    // fontWeight: 'bold',
+    fontSize: 16,
+    color: '#555',
+    marginVertical: 7,
+  },
+  horizontalLineGray: {
+    borderBottomColor: '#D0D0D0',
+    borderBottomWidth: 1,
+    marginTop: 10,
+    marginBottom: 10,
+  },
+});
```

Comment:
let's move this as well to a `index.style.js`

Reply:
moved them all to same index.style.js